phoenix-core/src/it/java/org/apache/phoenix/end2end/MapReduceIT.java

```diff
@@ -135,6 +164,73 @@ private void createAndTestJob(Connection conn, String s, double v, String tenant
 
     }
 
+    private void createPagedJobAndTestFailedJobDueToTimeOut(Connection conn, String s, double v, String tenantId,
```

Comment:
nit: function-level variables should have meaningful names more than 1 character (not "s" or "v"). (For temp variables in a loop this would be fine)